### PR TITLE
Add project guideline

### PR DESCRIPTION
**Pull request by user512**
Follow up on Notabli retro, @cjerozal drafted a project guideline and it seems to be great to publish publicly in markdown format for less Git savvy person to edit in the future. 
Screen Shot 2020-05-05 at 10 54 42 AM

### CODE REVIEW

File: src/_includes/layouts/default.njk [hunk 1]
```diff
@@ -27,6 +27,7 @@
           <li><a href="/code-of-conduct">Code of Conduct</a></li>
           <li><a href="/financial-reports">Financial Reports</a></li>
           <li><a href="/meeting-notes">Meeting Notes</a></li>
+          <li><a href="/project-guideline">Project Guideline</a></li>
```
Comment (zspencer) [inline src/_includes/layouts/default.njk:30]: I believe this should be plural, I.e. `project-guidelines`.

I also think that this isn't important enough in the [information hierarchy](https://www.bridgewaterlearning.co.za/2013/04/16/design-principles-hierarchy-of-information/) to be it's own top-level page, and maybe it belongs within the `/maintaining` page that I am sprouting in #28 ?

File: src/project-guideline/index.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+layout: layouts/default.njk
+---
+
+# Project Guideline
+## What?
+This is a list of guidelines to consider following for Zinc projects. It was initially created using thoughts/ideas from our Notabli Retrospective, but ideally will become a living document, contributed to over time.
```
Comment (zspencer) [inline src/project-guideline/index.md]: nit-pick, I don't think we need to share the history, but maybe we want to share the intent. I.e. "We've learned by doing that there are some things that make new projects flow more smoothly. This is a living document that will change over time, and was last updated on May 5th, 2020 "

**Review by zspencer — APPROVED** [review]
This looks great! I dropped some feedback that is non-blocking-but-may-be-worth-doing. If you want, I can merge it and do a follow-up patch to move it into the `/maintaining` directory once that gets merged.

File: src/_includes/layouts/default.njk (hunk 1 above)
Comment (maximegalon5) [reply src/_includes/layouts/default.njk:30]: I agree it should be plural

**Pull request merged**